packages/ecs-init/0001-bottlerocket-remove-interaction-with-Nvidia-lib.patch

```diff
@@ -0,0 +1,333 @@
+From 98f7025e054be506d431501422a74ba135fe6644 Mon Sep 17 00:00:00 2001
```

Comment:
If you haven't done so already, please open an issue with the debugging that's already been done with relevant snippets of conversation and logged output.

Reply:
There's now a comment in the spec that points to what I believe is the cause.  Per our offline discussion, however, I'm planning to move away from using ecs-init.  I may need a similar patch for the ECS agent's source code and can either annotate directly in the patch/spec, or can open an issue at that time.